core/src/main/java/nl/nn/adapterframework/core/Adapter.java

```diff
@@ -343,10 +340,7 @@ public Message formatErrorMessage(String errorMessage, Throwable t, Message orig
 		try {
 			Message formattedErrorMessage= errorMessageFormatter.format(errorMessage, t, objectInError, originalMessage, messageID, receivedTime);
 
-			if(msgLog.isEnabled(MSGLOG_LEVEL_TERSE)) {
-				String resultOrSize = (isMsgLogHidden()) ? "SIZE="+getFileSizeAsBytes(formattedErrorMessage) : formattedErrorMessage.toString();
-				msgLog.log(MSGLOG_LEVEL_TERSE_EFF, "formatted errormessage, result [{}]", resultOrSize);
-			}
+			logToMessageLogWithMessageContentsOrSize(Level.INFO, "Error", "result", formattedErrorMessage, null);
```

Comment:
Je logt nu alleen `msgLog.log(level, logMessage);` in dit geval `"Error"`. Moet dat niet iets beschrijvenders worden? Heb je een voorbeeldje hoe het er uit komt te zien?